lib/tc.py

```diff
@@ -50,6 +50,63 @@ def get_template_id(self, template_name):
         except APIError as error:
             raise GuardError(error)
 
+    def get_role_id(self, template_name, permission):
+        """
+        Returns a role id from a template permission combination
+        Args:
+            template_id (int): the id of the template
+            permission (str): permission for the role
+
+        Retruns:
+            (int): role id if found
+
+        Raises:
+            GuardError
+        """
+        api = self.api
+        try:
+            data = api.role_data()
+            # now iterate through all roles and search for the right combination of template id and permission
+            for role in data['results']:
+                role_type = role['name'].lower()
+                resource_type = role['summary_fields'].get('resource_type')
+                resource_name = role['summary_fields'].get('resource_name')
+                if ((resource_type) and
```

Comment:
what about creating a new function that returns only 'job template' results?
I'm trying to find a way to simplify this 4 lines if